bot/exts/moderation/infraction/_utils.py

```diff
@@ -298,3 +300,26 @@ async def send_private_embed(user: MemberOrUser, embed: discord.Embed) -> bool:
             "The user either could not be retrieved or probably disabled their DMs."
         )
         return False
+
+
+class StaffBanConfirmationView(interactions.ViewWithUserAndRoleCheck):
+    """The confirmation view that is sent when a moderator attempts to ban a staff member."""
+
+    confirmed = True
```

Comment:
Could we flip this, and make the default `False`, setting it to True when confirmed.

Shouldn't be an issue now, it jsut means that it fails safe if we have a bug around her ein teh future.

Reply:
I'm a big fan of defensive programming, stgm. Thanks!